addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/LearnMorePanel.java

```diff
@@ -0,0 +1,196 @@
+            contentPanel.setBackground(Color.WHITE);
+            int formPanelY = 0;
+
+            ExtensionHelp extHelp = Control.getSingleton().getExtensionLoader().getExtension(ExtensionHelp.class);
+            boolean isHelpAvailable = extHelp != null && extHelp.isHelpAvailable();
+
+            File guide = new File(Constant.getZapHome() + File.separator + "lang" + File.separator
+                    + Constant.messages.getString("gettingStarted.file"));
+
+            if (isHelpAvailable || guide.canRead()) {
+
+                contentPanel.add(new JLabel(Constant.messages.getString("quickstart.links.local")),
+                        LayoutHelper.getGBC(0, formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+                contentPanel.add(new JLabel(""),
+                        LayoutHelper.getGBC(2, formPanelY, 1, 1.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5))); // Spacer
+
+                if (guide.canRead()) {
+                    JLabel qsLabel = ulJLabel(Constant.messages.getString("quickstart.link.startguide"));
+                    qsLabel.setIcon(ExtensionQuickStart.PDF_DOC_ICON);
+                    qsLabel.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
+                    qsLabel.addMouseListener(new MouseAdapter() {
+                        public void mouseClicked(MouseEvent e) {
+                            try {
+                                Desktop.getDesktop().open(guide);
+                            } catch (Exception ex) {
+                                View.getSingleton().showWarningDialog(LearnMorePanel.this, Constant.messages
+                                        .getString("quickstart.link.warning.nostartguide", ex.getMessage()));
+                            }
+                        }
+                    });
+                    contentPanel.add(qsLabel,
+                            LayoutHelper.getGBC(1, ++formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+                }
+
+                if (isHelpAvailable) {
+                    JLabel helpLabel = ulJLabel("quickstart.link.userguide");
+                    helpLabel.setIcon(ExtensionHelp.getHelpIcon());
+                    helpLabel.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
+                    helpLabel.addMouseListener(new MouseAdapter() {
+                        public void mouseClicked(MouseEvent e) {
+                            ExtensionHelp.showHelp();
+                        }
+                    });
+                    contentPanel.add(helpLabel,
+                            LayoutHelper.getGBC(1, ++formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+                }
+            }
+
+            contentPanel.add(new JLabel(Constant.messages.getString("quickstart.links.online")),
+                    LayoutHelper.getGBC(0, ++formPanelY, 1, 0.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+
+            if (!guide.canRead()) {

```

Comment:
We should add a TODO somewhere (an issue?) to make this dynamic (e.g. when the add-on is un/installed). Same for the help.